frontend/src/features/projects/dispose/components/columns.tsx

```diff
@@ -37,6 +37,7 @@ const EditableClassificationCell = (cellInfo: any) => {
   return (
     <FastSelect
       formikProps={context}
+      placeholder="Must selct one"
```

Comment:
To prevent it from defaulting to `Surplus Active`

Reply:
/nit: fix typo "Must sel(e)ct one"

Reply:
oops! Thank you

Reply:
I am going to do some more playing around, this right now overrides the existing and only shows SE/SA

Reply:
Now that I understand the "full" business requirements which are (correct me if I'm wrong please):

> - Show current assigned value in "read-only" mode
> - Only display SE and SA as new options to select

I think this should warrant a new component that takes an initial "read-only" value object (with label and id to not loose the selection made before) and the allowed options. I wouldn't change the current Select component but create a new one for this specific use case

Reply:
@asanchezr , I think that might be best too. The only other ways I was thinking of was to add the optional attribute of `disabled` onto the type `SelectOption` and disable everything that isn't SA/SE. But the problem with that is it still shows them in the list - they are just grayed out. The other way I was thinking was to make the placeholder the previous value..